src/index.js

```diff
@@ -0,0 +1,173 @@
+//BackgroundColor and landscape will change based on temp ranges:
+// 95+	Red
+// 80-95	Orange
+// 70-80	green
+// 55-70	teal
+// 55 or below	whitish
+
+const state = {
+  temp: 60,
+};
+
+const updateTempBackground = () => {
+  const landscape = document.querySelector('#landscapeContainer');
+  if (state.temp >= 90) {
+    document.getElementById('temp').style.backgroundColor = 'red';
```

Comment:
Ideally this would set a class name of something like `hot` or `scorching` instead of directly setting the background color.

This would give us flexibility to change up the color scheme in the stylesheet since we'd then be able to do things like change the temperatures so that they go red to purple to blue.